src/utility/FileSystem.ts

```diff
@@ -4,4 +4,4 @@
 import { promisify } from "util";
 import * as rimraf from "rimraf";
 
-export const rimrafAsync = promisify(rimraf);
\ No newline at end of file
+export const rimrafAsync = promisify(rimraf.sync);
```

Comment:
Could you please provide context for this change? Why do we need to use the sync version of rimraf here?